cmd/gen-go-sample/testdata/sample_response_map_field.want

```diff
@@ -0,0 +1,60 @@
+// Copyright 2018 Google LLC
+//
```

Comment:
Is this file meant to be part of this PR? Doesn't seem to deal with CLI for sample configs....

Reply:
OH NO! Must be added here accidentally during git rebase. Thanks for catching this!